SMARTAGRIFOOD/vineyardData/examples/example-normalized.json

```diff
@@ -1,36 +1,125 @@
-{
-	"id": "VB638235",
-	"type": "AgriVineyardBlock",
-	"name": {
-		"value": "Northern block",
-		"type": "String"
-	},
-	"dateModified": {
-		"value": "2021-09-29T00:00:00.0Z",
-		"type": "DateTime"
-	},
-	"inclination": {
-		"value": {
-			"aspect": {
-				"value": "E",
-				"type": "String"
-			},
-			"slope": {
-				"value": "2",
-				"type": "String"
-			}
-		}
-	},
-	"rowSpacing": {
-		"value": 2,
-		"type": "Float"
-	},
-	"plantingStyle": {
-		"value": "rows",
-		"type": "String"
-	},
-	"dateCreated": {
-		"value": "2015-02-28T00:00:00.0Z",
-		"type": "DateTime"
-	}
-}
+{
+	"id": "VB638235",
+	"type": "AgriVineyardBlock",
+	"name": {
+		"value": "Northern block",
+		"type": "String"
+	},
+	"coordinates": {
+		"value": "[123,234,345]",
+		"type": "String"
+	},
+	"address": {
+		"value": "Orehovci 1a, 9250 Gornja Radgona, Slovenia",
+		"type": "String"
+	},
+	"size": {
+		"value": 3014,
+		"type": "Float"
+	},
+	"numVinesRepresented": {
+		"value": 237,
+		"type": "Integer"
+	},
+	"plants": {
+		"value": {
+			"trainingSystem": {

```

Comment:
sure about the name 'trainingSystem' or 'trailingSystem' ?

Reply:
training system is correct